projections/weather/lib/weather/shapes.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module Weather
+  module Shapes
+    include Smithy::Client::Shapes
+
+    CityCoordinates = StructureShape.new(id: "example.weather#CityCoordinates", traits: {})
+    CityId = StringShape.new(id: "example.weather#CityId", traits: {"smithy.api#pattern"=>"^[A-Za-z0-9 ]+$"})
+    CitySummaries = ListShape.new(id: "example.weather#CitySummaries", traits: {})
+    CitySummary = StructureShape.new(id: "example.weather#CitySummary", traits: {"smithy.api#references"=>[{"resource"=>"example.weather#City"}]})
+    GetCityInput = StructureShape.new(id: "example.weather#GetCityInput", traits: {"smithy.api#input"=>{}})
+    GetCityOutput = StructureShape.new(id: "example.weather#GetCityOutput", traits: {"smithy.api#output"=>{}})
+    GetCurrentTimeOutput = StructureShape.new(id: "example.weather#GetCurrentTimeOutput", traits: {"smithy.api#output"=>{}})
+    GetForecastInput = StructureShape.new(id: "example.weather#GetForecastInput", traits: {"smithy.api#input"=>{}})
+    GetForecastOutput = StructureShape.new(id: "example.weather#GetForecastOutput", traits: {"smithy.api#output"=>{}})
+    ListCitiesInput = StructureShape.new(id: "example.weather#ListCitiesInput", traits: {"smithy.api#input"=>{}})
+    ListCitiesOutput = StructureShape.new(id: "example.weather#ListCitiesOutput", traits: {"smithy.api#output"=>{}})
+    NoSuchResource = StructureShape.new(id: "example.weather#NoSuchResource", traits: {"smithy.api#error"=>"client"})
+
+    CityCoordinates.add_member("latitude", Float, {"smithy.api#required"=>{}})
+    CityCoordinates.add_member("longitude", Float, {"smithy.api#required"=>{}})
+    CityCoordinates.type = Types::CityCoordinates
+
```

Comment:
I could omit these whitelines tbh

Reply:
Its only one line in between these right?  I think the seperation is useful since its effectively a different section

Reply:
Whiteline is fine.